app/controllers/concerns/idv_session.rb

```diff
@@ -8,6 +8,7 @@ def confirm_idv_session_started
   def confirm_idv_attempts_allowed
     if idv_attempter.exceeded?
       flash[:error] = t('idv.errors.hardfail')
+      analytics.track_event(Analytics::IDV_INITIAL, success: false, idv_attempts_exceeded: true)
       redirect_to verify_fail_url
```

Comment:
This analytics event here should be named something like `IDV_VISIT_AFTER_MAX_ATTEMPTS`, and should pass in the current request path as its attribute because it can be captured on any of the 5 endpoints that can trigger this `before_action`. If someone who has already exceeded attempts tries to visit any IDV endpoint that includes the `confirm_idv_attempts_allowed` callback, then this event will be triggered, so we need a way to differentiate between where/how the event was triggered. We don't need the `success` and `idv_attempts_exceeded` attributes in this case because this can only be triggered if the attempts have already been exceeded.